doc/sphinx/addendum/type-classes.rst

```diff
@@ -86,20 +85,19 @@ use alternatively the :attr:`program` attribute to get richer
 facilities for dealing with obligations.
 
 
-Binding classes
----------------
+Binding Typeclasses
```

Comment:
```suggestion
Binding typeclasses
```

Generally we've been capitalizing only the first word of section titles.  Maybe we'll change that at some point, but consistency is a good idea.  (comment applies in multiple places)

Reply:
I couldn't remember if we used title case in titles or not, I'll change them back.